### PR TITLE
Testing: part 2

### PR DESCRIPTION
**Pull request by rex-9**
## Implementation
- Used jest framework for testing.
- Created a test file(s) ([..].test.js) for a file(s) containing the following functions:
. a function for editing the task description.
. a function for updating an item's 'completed' status.
. the "Clear all completed" function.
- Used the mock storage object you have created in Part 1 of this project to mock the localStorage updates.
- Mocked the HTML elements to test DOM manipulation functions.
- Made sure tests using the describe() method.

### CODE REVIEW

File: src/test/crud.test.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import {
+  addTask,
+  editTask,
+  toggleTask,
+  removeTask,
```
Comment (FrederickMih) [inline src/test/crud.test.js]: - [ ] Based on the requirements of the project, you are supposed to test the `Clear all completed` function. Kindly add a test for that function and move a step toward approval.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @HtetNaing0814 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️

- [ ] kindly update the Author section of the README file to contain all the members of this project. Let's also include LinkedIn, and Twitter links so that your information is open to recruiters.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/test/crud.test.js (hunk 1 above)
Comment (rex-9) [reply src/test/crud.test.js]: Bro, remove task is clear all completed tasks. It removes all the items that have "completed" false. You should've checked the function, bro. You can't make a review by only just looking at the function names.

**Comment by rex-9** [comment]
Thank u so much for your review of the readme file.

**Review by roshan-bajgain — APPROVED** [review]
Hi @HtetNaing0814, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- PR has a good title and summary. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/test/crud.test.js (hunk 1 above)
Comment (Godfrey-Alimony) [reply src/test/crud.test.js]: Thank you so much

**Pull request merged**